recipes/ligo.skymap/meta.yaml

```diff
@@ -0,0 +1,159 @@
+{% set name = "ligo.skymap" %}
+{% set version = "0.0.19" %}
+{% set sha256 = "a2a5bac2b56dfce41261b6644709760bcdc7b59b06202f9f93d180ae968a19e7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # tell ah_bootstrap to not download anything
+    - setup.cfg-offline.patch
+    # fix requirements
+    - requirements.patch
+
+build:
+  number: 0
+  skip: true  # [win or py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv --install-option=--use-system-libraries"
+  entry_points:
+    - bayestar-localize-coincs = ligo.skymap.tool.bayestar_localize_coincs:main
+    - bayestar-localize-lvalert = ligo.skymap.tool.bayestar_localize_lvalert:main
+    - bayestar-mcmc = ligo.skymap.tool.bayestar_mcmc:main
+    - bayestar-realize-coincs = ligo.skymap.tool.bayestar_realize_coincs:main
+    - bayestar-sample-model-psd = ligo.skymap.tool.bayestar_sample_model_psd:main
+    - ligo-skymap-combine = ligo.skymap.tool.ligo_skymap_combine:main
+    - ligo-skymap-contour = ligo.skymap.tool.ligo_skymap_contour:main
+    - ligo-skymap-from-samples = ligo.skymap.tool.ligo_skymap_from_samples:main
+    - ligo-skymap-plot = ligo.skymap.tool.ligo_skymap_plot:main
+    - ligo-skymap-plot-airmass = ligo.skymap.tool.ligo_skymap_plot_airmass:main
+    - ligo-skymap-plot-stats = ligo.skymap.tool.ligo_skymap_plot_stats:main
+    - ligo-skymap-plot-volume = ligo.skymap.tool.ligo_skymap_plot_volume:main
+    - ligo-skymap-stats = ligo.skymap.tool.ligo_skymap_stats:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy 1.14.*
```

Comment:
This pin would exclude version 1.15 of numpy IIUIC. Did you mean to do that? A simple `numpy >=1.14` would be better for maintenance.

Reply:
The way I understand is that this will guarantee that the _build_ runs on numpy-1.14, but the runtime pin is just numpy>=1.14,<2.0.0a0.

If I just use `numpy >=1.14` in `requirements/host` then the build step chooses the latest version of numpy, and the pin for `requirements/run` then uses that version (1.15.4?) as the minimum value.

I could quite easily have misunderstood the way this works.

Reply:
Whoops! You are right! Sorry!